uplink/config.go

```diff
@@ -28,8 +28,8 @@ type EncryptionConfig struct {
 	EncryptionKey     string `help:"the root key for encrypting the data which will be stored in KeyFilePath" setup:"true"`
 	KeyFilepath       string `help:"the path to the file which contains the root key for encrypting the data"`
 	EncAccessFilepath string `help:"the path to a file containing a serialized encryption access"`
-	DataType          int    `help:"Type of encryption to use for content and metadata (1=AES-GCM, 2=SecretBox)" default:"1"`
-	PathType          int    `help:"Type of encryption to use for paths (0=Unencrypted, 1=AES-GCM, 2=SecretBox)" default:"1"`
+	DataType          int    `help:"Type of encryption to use for content and metadata (2=AES-GCM, 3=SecretBox)" default:"2"`
+	PathType          int    `help:"Type of encryption to use for paths (1=Unencrypted, 2=AES-GCM, 3=SecretBox)" default:"2"`
```

Comment:
They are changed to reflect to what it was before ie AESGCM(before the renaming the default value is 1). When renamed the defaults changed to 2 instead of 1 because after renaming a new enum called 'unspecified' was there as '0'.